### PR TITLE
aligned examples, extended description, added alternatives

### PR DESCRIPTION
**Pull request by tlodderstedt**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,11 +27,144 @@ Notes:
...
+        "given_name": null,
+        "family_name": null,
+        "birthdate": null
+    }
+}
+```
+
+### claims parameter (with identity assurance requirements)
+
+The TP may also request claims fulfilling the requirtements of a certain trust framework and identity assurance level. It does so by adding a `verified_claims` claims element to the `vp_token` section, which defines the verification requirements and lists the respective End-User claims.
```
Comment (AdamJLemmon) [inline README.md]: `The TP` => RP?

File: README.md [hunk 2]
```diff
@@ -48,21 +181,33 @@ The following is a non-normative example of a W3C VC using the external [JWT pro
...
+        "country":"SE"
     }
   }
 }
 ```
 
 ## W3C Verifiable Credential using internal proofs
 
-The following is a non-normative example of a W3C VC using the internal proof format. The proof property contains a JSON-LD Proof and uses the detached JWS encoding for the signature representation:
+The following is a non-normative example of a W3C VC using the internal proof format. The proof property contains a JSON-LD Proof and uses the detached JWS encoding for the signature representation.
```
Comment (AdamJLemmon) [inline README.md:209]: Not sure is we want to align the VC spec's use of `external` and `embedded` as opposed to `internal` ?

https://www.w3.org/TR/2019/REC-vc-data-model-20191119/#proofs-signatures

File: README.md [hunk 3]
```diff
@@ -27,11 +27,144 @@ Notes:
...
+
+```
+{
+    "id_token": {
+        "acr": null
+    },
+    "vp_token": {
+        "verified_claims": {
+            "verification": {
+                "trust_framework": {
```
Comment (awoie) [inline README.md:80]: @tlodderstedt In that case the `eidas` trust framework would define which `issuer`DIDs are trustworthy. Is this correct?

File: README.md [hunk 4]
```diff
@@ -48,21 +181,33 @@ The following is a non-normative example of a W3C VC using the external [JWT pro
...
+      "birthdate": "1949-01-22",
+      "place_of_birth": {
+        "country": "SE",
+        "locality": "Örnsköldsvik"
+      },
+      "nationality": "SE",
+      "number": "4901224131",
+      "date_of_issuance":"2010-03-23",
+      "date_of_expiry":"2020-03-22"
+      "issuer":{
```
Comment (awoie) [inline README.md]: @tlodderstedt the `issuer` property should be one level above.

File: README.md [hunk 5]
```diff
@@ -73,15 +218,27 @@ The following is a non-normative example of a W3C VC using the internal proof fo
...
+    "birthdate": "1949-01-22",
+    "place_of_birth": {
+       "country": "SE",
+       "locality": "Örnsköldsvik"
+    },
+     "nationality": "SE",
+    "number": "4901224131",
+    "date_of_issuance":"2010-03-23",
+    "date_of_expiry":"2020-03-22"
+    "issuer":{
```
Comment (awoie) [inline README.md]: @tlodderstedt the `issuer` property should be one level above. In that case likely merged with the `issuer` property.

**Review by awoie — CHANGES_REQUESTED** [review]
LGTM in general but I had one question and I think the `issuer` property needs to be fixed in some of the examples.

File: README.md (hunk 2 above)
Comment (tlodderstedt) [reply README.md:209]: @awoie WDYT?

File: README.md (hunk 3 above)
Comment (tlodderstedt) [reply README.md:80]: yes. There is a need for an authoritative list of such issuers similar to https://ec.europa.eu/cefdigital/wiki/display/EIDCOMMUNITY/Overview+of+pre-notified+and+notified+eID+schemes+under+eIDAS and https://webgate.ec.europa.eu/tl-browser/. ESSIF seems to be heading there.

File: README.md (hunk 5 above)
Comment (tlodderstedt) [reply README.md]: done

Comment (tlodderstedt) [reply README.md]: it seems to make sense here.

File: README.md (hunk 4 above)
Comment (tlodderstedt) [reply README.md]: just as a claim within the vc container?

Comment (tlodderstedt) [reply README.md]: done

File: README.md (hunk 2 above)
Comment (awoie) [reply README.md:209]: let's fix that later. i agree that embedded and external are the correct terms.

**Pull request merged**